backend/de.metas.business/src/main/java/de/metas/order/model/interceptor/C_Order.java

```diff
@@ -230,17 +230,20 @@ public void setDeliveryViaRule(final I_C_Order order)
 	@CalloutMethod(columnNames = I_C_Order.COLUMNNAME_C_BPartner_ID)
 	public void setPaymentRule(final I_C_Order order)
 	{
-		final I_C_BPartner bpartner = order.getC_BPartner();
-		final PaymentRule paymentRule;
-		if (bpartner != null && bpartner.getPaymentRule() != null)
-		{
-			paymentRule = PaymentRule.ofCode(bpartner.getPaymentRule());
-		}
-		else
+		if (!InterfaceWrapperHelper.isCopying(order))
```

Comment:
shouldn't we then have this if also in `backend/de.metas.business/src/main/java/de/metas/invoice/interceptor/C_Invoice.java` ?

Reply:
shouldn't we also have `&& !InterfaceWrapperHelper.isNew(invoice)` here?

Reply:
If I understood correctly from Daniela::

- A new order is supposed to inherit the payment rule from the BPartener:
- A new invoice is supposed to inherit the payment rule from the related order.

so i think we shouldn't add it to `de.metas.order.model.interceptor.C_Order`